site/content/faq/can-zap-be-used-with-a-screenreader.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Can ZAP be used with a Screenreader?"
+type: faq
+category: Desktop UI
+weight: 7
+---
+
+ZAP, as well as its add-ons, uses Java Swing for it's user interface. For screenreader users, this means that even though a lot of the user interface components technically work, the experience could be better in places.
```

Comment:
`it's` → `its`, I didn't check all text yet (there might be other typos).

Reply:
Good catch